stable/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml

```diff
@@ -50,8 +50,8 @@ spec:
             - --retry-interval-start=1s
             - --retry-interval-max=5m
             - --worker-threads=20
-            - "-leader-election"
-            - "-leader-election-type=leases"
+            - -leader-election
```

Comment:
doesn't it need to be double-dash ?

Reply:
look at https://github.com/kubernetes-sigs/azuredisk-csi-driver for the next upgrade

Reply:
I will ignore the inconsistency of the also in upstream existing arguments 🤣 ... for now 😉